src/components/Project/ImageAnalysis/ImageViewer.vue

```diff
@@ -23,6 +28,7 @@ let imageBounds
 let initialCenter = [0, 0]
 let initialZoom = 1
 let lastDrawnLine = null
+let LayerControl = null
```

Comment:
minor thing, I would name this `layerControl` to keep consistency

Reply:
oopsies good catch